pkg/test/framework/scope.go

```diff
@@ -71,31 +71,12 @@ func (s *scope) done(nocleanup bool) error {
 			}
 		}
 	}
-	s.resources = nil // set resources to nil to avoid resetting them
-
-	if e := s.reset(); e != nil {
-		err = multierror.Append(err, e)
-	}
+	s.resources = nil
 
 	scopes.Framework.Debugf("Done cleaning up scope: %v", s.id)
 	return err
 }
 
-func (s *scope) reset() error {
-	var err error
-	for _, r := range s.resources {
-		if res, ok := r.(resource.Resetter); ok {
-			scopes.Framework.Debugf("Resetting resource: %s", r.ID())
-			if e := res.Reset(); e != nil {
```

Comment:
This is the call-site.

Reply:
ok, but it shouldn't have been called prior to this PR.  The only call site was from `scope.done()`, which nils out the resources before calling it.